tests/features/asset_release/add_asset_release.feature

```diff
@@ -38,7 +39,7 @@ Feature: "Add release" visibility options.
     And I click "Add release"
     Then I should see the heading "Add Release"
     And the following fields should be present "Name, Release number, Release notes, Documentation, Spatial coverage, Keyword, Status, Language"
-    And the following field widgets should be present "Contact information, Owner"
+    And the following field widgets should be present "Contact information"
```

Comment:
I removed the _Owner_ field from asset_release form, in this PR,  because we sync the owner from the parent solution.